fapolicy_analyzer/ui/subject_list.py

```diff
@@ -106,29 +106,21 @@ def _columns(self):
         return [trustColumn, accessColumn, fileColumn]
 
     def _trust_markup(self, subject):
-        u_str = "U"
-        at_str = "AT"
-        st_str = "ST"
-
-        status = (
-            dbtrust.status.lower()
-            if (dbtrust := self.__find_db_trust(subject))
-            else None
-        )
 
-        if subject.trust.lower() == "at":
-            if not status or status == "d":
-                at_str = '<span color="red"><b>AT</b></span>'
-            else:
-                at_str = '<span color="green"><u><b>AT</b></u></span>'
-        elif subject.trust.lower() == "st":
-            if not status or status == "d":
-                st_str = '<span color="red"><b>ST</b></span>'
-            else:
-                st_str = '<span color="green"><u><b>ST</b></u></span>'
-        elif subject.trust.lower() == "u":
-            if not status or status == "d":
-                u_str = '<span color="green"><u><b>U</b></u></span>'
+        status = subject.trust_status.lower()
+
+        if status == "u":
```

Comment:
I think this should be `status == 'd'` right?

Reply:
When I switched to using trust status it was always a "u" or "t". Was something changed when `trust_status` was added?

Reply:
According to this guy https://github.com/ctc-oss/fapolicy-analyzer/blob/ea154174a5b9b94ce1253af45756e71151d3d2b5/crates/analyzer/src/events/analysis.rs#L94 the `u` is for unknown file trust and `d` is for discrepancy. I'm not exactly sure how a file could be in one of the trust databases and still have a `u` for status, but maybe the best approach would would be to color it red if it's status is `not 't'`.

Reply:
Good call. Fixed.